waggle-dance-api/pom.xml

```diff
@@ -88,6 +88,13 @@
       <groupId>commons-lang</groupId>
       <artifactId>commons-lang</artifactId>
     </dependency>
+    
+    <!-- SSH library -->
+    <dependency>
+      <groupId>com.hotels</groupId>
+      <artifactId>hcommon-hive-metastore</artifactId>
```

Comment:
ah we need this in core and api, better we make this a managed dependency in the parent pom and remove the version here and in the core/pom.xml